FirstProject/assets/evaluation.less

```diff
@@ -74,19 +98,15 @@ table tr th {
   margin-top: 120px;
 }
 
-.col-md-4 {
```

Comment:
bootstrap? :)

Reply:
Initially was bootstrap, but I removed it.
![eval](https://user-images.githubusercontent.com/31060543/29771662-bc761f9a-8bfd-11e7-9a48-8d2fae89465b.png)